docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/customizing-policy.adoc

```diff
@@ -0,0 +1,28 @@
+= Customizing Policy
+
+An Enterprise Contract Policy defines which checks are performed by EC during integration tests and
+release. If EC reports violations, the best course of action, in most cases, is to adjust the build
+process to remedy the violation. For example, if EC reports that a certain required scanner was not
```

Comment:
Should we indicate that the policy violation should include a hint for how to resolve the issue? I know that we describe that later, but just asking if you want to reference that now as well.

Reply:
Added:

> Read the text associated with the violation as that often provides context-specific information to help remedy violations.